draft-irtf-cfrg-vdaf.md

```diff
@@ -2219,11 +2228,11 @@ The sharding algorithm involves the following steps:
 
 1. Encode the Client's measurement for the FLP
 2. Shard the measurement into a sequence of measurement shares
```

Comment:
I think we need a more gentle introduction to the idea of multi-proofs. Here I think we should speak just in terms of a single proof. Then in a short paragraph below, recall the `PROOFS` parameter and say what it does.